### PR TITLE
draft2

### PR DESCRIPTION
**Pull request by vlerenc**
title:ggg

### CODE REVIEW

**Comment by gardener-botmanager** [comment]
@vlerenc Command "/cla" failed with "KeyError('pull-request')".
<details>
<summary>Additional Information</summary>

```
Redacted in public. Check backend logs.
```
</details>

**Comment by gardener-botmanager** [comment]
@vlerenc Label "cla" does not exist for this repository.

**Comment by gardener-botmanager** [comment]
@vlerenc Command "/cla" failed with "TypeError('can only concatenate str (not "int") to str')".
<details>
<summary>Additional Information</summary>

```
Redacted in public. Check backend logs.
```
</details>

**Comment by gardener-botmanager** [comment]
@vlerenc Label "cla" does not exist for this repository.

**Comment by gardener-botmanager** [comment]
@vlerenc Label "cla" does not exist for this repository.

**Comment by gardener-botmanager** [comment]
@vlerenc Command "/cla" failed with "OSError('WORKED')".
<details>
<summary>Additional Information</summary>

```
Redacted in public. Check backend logs.
```
</details>

**Comment by gardener-botmanager** [comment]
@vlerenc Label "cla" does not exist for this repository.

**Comment by gardener-botmanager** [comment]
@vlerenc I reached successfully out to the cla-assistant to recheck this pull request.

**Comment by gardener-botmanager** [comment]
@vlerenc Label "cla" does not exist for this repository.

**Comment by vlerenc** [comment]
sss

**Comment by gardener-botmanager** [comment]
@vlerenc Label "dummy" does not exist for this repository.

**Comment by gardener-botmanager** [comment]
@vlerenc Label "ss" does not exist for this repository.

**Comment by gardener-botmanager** [comment]
@vlerenc Command /ddd not understood.

**Comment by gardener-botmanager** [comment]
@vlerenc Label "ddd" does not exist for this repository.

**Comment by gardener-botmanager** [comment]
@vlerenc Command /dsfd not understood.

**Comment by gardener-botmanager** [comment]
@vlerenc Command "/dsfd" failed with "NameError("name 'dsfdsf' is not defined")".
<details>
<summary>Additional Information</summary>

```
Redacted in public. Check backend logs.
```
</details>

**Comment by gardener-botmanager** [comment]
@vlerenc Label "dsfd" does not exist for this repository.

**Comment by gardener-botmanager** [comment]
@vlerenc I reached successfully out to the cla-assistant to recheck this pull request.

**Comment by gardener-botmanager** [comment]
@vlerenc Sure, if you insist, I can call you a dummy.

**Comment by gardener-botmanager** [comment]
@vlerenc Label "cla" does not exist for this repository.

**Comment by gardener-botmanager** [comment]
@vlerenc Label "dummy" does not exist for this repository.

**Comment by gardener-botmanager** [comment]
@vlerenc Command "/dummy" failed with "TypeError('isinstance() arg 2 must be a type or tuple of types')".
<details>
<summary>Additional Information</summary>

```
Redacted in public. Check backend logs.
```
</details>

**Comment by gardener-botmanager** [comment]
@vlerenc Label "dummy" does not exist for this repository.

**Comment by gardener-botmanager** [comment]
@vlerenc Command "/cla" failed with "TypeError('isinstance() arg 2 must be a type or tuple of types')".
<details>
<summary>Additional Information</summary>

```
Redacted in public. Check backend logs.
```
</details>

**Comment by gardener-botmanager** [comment]
@vlerenc Label "cla" does not exist for this repository.

**Comment by gardener-botmanager** [comment]
@vlerenc Sure, if you insist, I can call you a dummy.

**Comment by gardener-botmanager** [comment]
@vlerenc Label "dummy" does not exist for this repository.

**Comment by gardener-botmanager** [comment]
@vlerenc Sure, if you insist, I can call you a dummy.

**Comment by gardener-botmanager** [comment]
@vlerenc Sure, if you insist, I can call you a dummy.

**Comment by gardener-botmanager** [comment]
@vlerenc I reached successfully out to the cla-assistant to recheck this pull request.

**Comment by gardener-botmanager** [comment]
@vlerenc Command "/cla" failed with "ConnectionError(MaxRetryError("HTTPSConnectionPool(host='cla-assistant.ioo', port=443): Max retries exceeded with url: /check/gardener-attic/bot-test-repo-1?pullRequest=29 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x109872510>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))"))".
<details>
<summary>Additional Information</summary>

```
Redacted in public. Check backend logs.
```
</details>

**Comment by gardener-botmanager** [comment]
@vlerenc I reached out successfully to the cla-assistant to recheck this pull request.

**Comment by gardener-botmanager** [comment]
@vlerenc Command "/invite" failed with "Review cannot be requested from pull request author.".
<details>
<summary>Additional Information</summary>

```
Redacted in public. Check backend logs.
```
</details>

**Comment by vlerenc** [comment]
/invite @gardener-botmanager @gardener-attic/team-botmanager , @gardener-ci

**Comment by vlerenc** [comment]
/invite @gardener-botmanager @gardener-attic/team-botmanager , @gardener-ci

File: CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,3 @@
+// Copyright
```
Comment (vlerenc) [inline CODEOWNERS:1]: dsdsfd

**Pull request closed without merging**